wordpress-mobile/WordPress-iOS
# UI Modernization: Me tab UI tweaks

https://github.com/wordpress-mobile/WordPress-iOS/pull/21348#issuecomment-1691527480

## Description
- Updates Me tab icon size to 28 pt, unselected image alpha to 36%
- Updates site icon size to 48pt
- Updates Me header view display name and username text colors

## How to test
- Switch to the My Site tab and verify the site icon size is 48pt
- Verify that the unselected icon for the Me tab has an alpha of 36%
- Switch to the Me tab
- Verify that the selected icon for the Me tab has full opacity
- Verify that the display name and username text color are updated to primary / secondary text colors

### Updated Me tab icon

<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/108b678e-f231-4065-a1ec-614008e851c2" width=200> | <img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/ab1db041-1c99-4160-aa63-a25393734320" width=200> 
-- | --

### Updated site icon size

Before | After
-- | --
<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/2eb02ce4-4547-4740-a26a-457aa021404a" width=200> | <img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/55c74c5c-942f-46f7-9bb5-bfebea9d81f4" width=200>


### Me screen display name / username text color
<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/754eb2d5-1d59-402d-95e1-a75362eb79b1" width=200> | <img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/8f59b8e7-25e8-4ed8-8386-91a46f01a00e" width=200> 
-- | --


## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:


UI Changes testing checklist:
